src/conn/conn_api.c

```diff
@@ -2671,6 +2671,18 @@ __conn_config_file_system(WT_SESSION_IMPL *session, const char *cfg[])
 #endif
         }
     }
+
+    /*
+     * Live restore leaves the state file on disk after live restore has completed, otherwise we'll
+     * run into issues when the user reopens a WiredTiger with the same live restore config but
+     * WiredTiger will assume it's a brand new live restore. To manage this we clean up the file the
+     * next time we open wiredtiger with a non-live restore config. This is expected to be done by
+     * the user immediately after live restore completes.
+     */
```

Comment:
I haven't created a FIXME to delete this file as part of taking a backup yet. I'm not sure if plan to pursue that